Cargo.toml

```diff
@@ -58,6 +58,7 @@ once_cell = "1.7.2"
 scroll = "0.10.2"
 target-lexicon = "0.12.0"
 pyproject-toml = "0.1.0"
+python-pkginfo = "0.3.2"
```

Comment:
I think python-pkginfo would also be a good candidate to move to the pyo3 organization.

---

You can remove the mailparse dependency from maturin since read_distribution.rs was the only place using it.

Reply:
Moved to https://github.com/PyO3/python-pkginfo-rs